firmware/ec/common/inc/ocmp_wrappers/ocmp_at45db.h

```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright (c) 2017-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*/
+
+#ifndef COMMON_INC_OCMP_WRAPPERS_OCMP_FLASH_H_
+#define COMMON_INC_OCMP_WRAPPERS_OCMP_FLASH_H_
+
+#define PAYLOAD_SIZE        47
```

Comment:
Suggest these be in alphabetical order

Reply:
As per our current coding guidelines the "#include"s are to be in alphabetical order . If the same needs to be extended for "#define"s as well then we need to change this as part of a separate request as a lot of files might need to be changed .